src/index.html

```diff
@@ -17,6 +17,11 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), BEM modifier classes should not be used on their own. Instead, use them alongside the block or element class that they modify, like `class="stopwatch stopwatch--speed-up"`.